packages/cdktf-cli/lib/get/generator/constants/provider-namespaces.ts

```diff
@@ -0,0 +1,614 @@
+    comment: "AWS Redshift",
+  },
+  resourcegroups: {
+    name: "ResourceGroups",
+    comment: "AWS Resource Groups",
+  },
+  route53: {
+    name: "Route53",
+    comment: "AWS Route 53",
+  },
+  s3: {
+    name: "S3",
+    comment: "AWS Simple Storage Service",
+  },
+  ses: {
+    name: "SES",
+    comment: "AWS Simple Email Service",
+  },
+  sns: {
+    name: "SNS",
+    comment: "AWS Simple Notification Service",
+  },
+  sqs: {
+    name: "SQS",
+    comment: "AWS Simple Queue Service",
+  },
+  ssm: {
+    name: "SSM",
+    comment: "AWS Systems Manager",
+  },
+  ssoadmin: {
+    name: "SSOAdmin",
+    comment: "AWS SSO Admin",
+  },
+  swf: {
+    name: "SWF",
+    comment: "AWS Simple Workflow Service",
+  },
+  sagemaker: {
+    name: "SageMaker",
+    comment: "AWS SageMaker",
+  },
+  secretsmanager: {
+    name: "SecretsManager",
+    comment: "AWS Secrets Manager",
+  },
+  securityhub: {
+    name: "SecurityHub",
+    comment: "AWS Security Hub",
+  },
+  serverlessapplicationrepository: {
+    name: "ServerlessApplicationRepository",
+    comment: "AWS Serverless Application Repository",
+  },
+  servicecatalog: {
+    name: "ServiceCatalog",
+    comment: "AWS Service Catalog",
+  },
+  service_discovery: {
+    name: "ServiceDiscovery",
+    comment: "AWS Service Discovery",
+  },
+  servicequotas: {
+    name: "ServiceQuotas",
+    comment: "AWS Service Quotas",
+  },
+  shield: {
+    name: "Shield",
+    comment: "AWS Shield Protection",
+  },
+  signer: {
+    name: "Signer",
+    comment: "AWS Signer",
+  },
+  simpledb: {
+    name: "SimpleDB",
+    comment: "AWS SimpleDB",
+  },
+  sfn: {
+    name: "SFN",
+    comment: "AWS Step Functions",
+  },
+  storagegateway: {
+    name: "StorageGateway",
+    comment: "AWS Storage Gateway",
+  },
+  synthetics: {
+    name: "Synthetics",
+    comment: "AWS Synthetics",
+  },
+  timestreamwrite: {
+    name: "TimestreamWrite",
+    comment: "AWS Timestream Write",
+  },
+  transfer: {
+    name: "Transfer",
+    comment: "AWS Transfer",
+  },
+  vpc: {
+    name: "VPC",
+    comment: "AWS VPC",
+    additionalPrefix: [
+      "customer_gateway",
+      "default_network_acl",
+      "default_route_table",
+      "default_security_group",
+      "default_subnet",
+      "default_vpc",
+      "egress_only_internet_gateway",
+      "flow_log",
+      "internet_gateway",
+      "main_route",
+      "nat_gateway",
+      "network",
+      "route_table",
+      "route",
+      "security_group",
+      "subnet",
+      "vpn",
+    ],
+  },
+  wafv2: {
+    name: "WAFV2",
+    comment: "AWS WAF V2",
+  },
+  wafregional: {
+    name: "WAFRegional",
+    comment: "AWS WAF Regional",
+  },
+  waf: {
+    name: "WAF",
+    comment: "AWS WAF",
+  },
+  worklink: {
+    name: "WorkLink",
+    comment: "AWS WorkLink",
+  },
+  workspaces: {
+    name: "Workspaces",
+    comment: "AWS WorkSpaces",
+  },
+  xray: {
+    name: "XRay",
+    comment: "AWS X-Ray",
+  },
+  _datasources: {
```

Comment:
This group is a bit strange. Some entries seem like rather generic non-service data sources, but others seem like they would fit within a service.

Reply:
I took the ones in the DataSources side bar entry on the provider